src/platform/bizhawk/base.mak

```diff
@@ -0,0 +1,130 @@
+CCFLAGS = \
+	-O3 -fomit-frame-pointer -fexpensive-optimizations -flto \
+	-I../../../../include -I../../.. -std=gnu11 \
+	-Wall -Wno-stringop-overflow -Wno-lto-type-mismatch \
+	-Wno-ignored-optimization-argument -Wno-unknown-warning-option \
+	-DMINIMAL_CORE=2 -DDISABLE_THREADING -DM_CORE_GBA \
+	-DCOLOR_16_BIT -DMGBA_STANDALONE -DENABLE_DEBUGGERS \
```

Comment:
I'm going to need to test if this compiles. I changed a bunch of the build flags somewhat recently, so this may not work anymore.